pkg/proxy/request.go

```diff
@@ -112,49 +109,49 @@ func fromHTTPRequestToLoadTestSpec(r *http.Request, logger *zap.Logger, allowedC
 	o, err := getOverwrite(r)
 	if err != nil {
 		logger.Debug("Bad value: ", zap.String("field", overwrite), zap.Bool("value", o), zap.Error(err))
-		return apisLoadTestV1.LoadTestSpec{}, fmt.Errorf("bad %q value: should be bool", overwrite)
+		return apisLoadTestV1.LoadTestSpec{}, fmt.Errorf("bad %s value: should be bool", overwrite)
```

Comment:
I personally like the quotes from `%q`, I think it makes more clear that the field name is not part of the string.. but the message could be improved too, something like:
 ```golang
 fmt.Errorf("field %q should have a boolean value", overwrite)
 ```

Reply:
I didn't like the slashes in the error message like this:
`
"error": "bad \"distributedPods\" value: should be integer"
`